romanNumerals.js

```diff
@@ -0,0 +1,23 @@
+function convertToOldRoman(num) {
+  var newNum = num
+  var roman = ""
+  var array = [{number: 1000, symbol: "M"},
+              {number: 500, symbol: "D"},
+              {number: 100, symbol: "C"},
+              {number: 50, symbol: "L"},
+              {number: 10, symbol: "X"},
+              {number: 5, symbol: "V"},
+              {number: 1, symbol: "I"}]
+
+  array.forEach(function (placeValue) {
```

Comment:
Very cool! Love the reusability of this with the ability to just continue to add to the `array` if we need higher numbers